### ISSUE TITLE
Registration:OP-Registration-Dynamic: status=ERROR, message=Missing required attribute 'error'

### ISSUE THREAD

**Issue by fcorneli** [openid-certification/oidctest#70]
We are playing around with the OpenID Connect certification web application and are receiving a weird error.
```
Registration:OP-Registration-Dynamic: status=ERROR, message=Missing required attribute ‘error'
```
Hence we return:
```
url:https://www.e-contract.be/openid-connect-test/register status_code:201
```
But the web application complains about a missing error attribute, this while our response is positive, and thus no error happened.

Maybe the error message is wrong?

**Comment by zandbelt** [comment]
I believe the root cause was addressed on the mailing list. It would be convenient to have the actual initial error show up in the logs rather than the derived one:
```
2017-11-24 09:59:44,820 otest.aus.tool:INFO <--<-- 2 --- Registration -->-->
2017-11-24 09:59:44,821 requests.packages.urllib3.connectionpool:INFO Starting new HTTPS connection (1): www.e-contract.be
2017-11-24 09:59:45,217 requests.packages.urllib3.connectionpool:DEBUG "POST /openid-connect-test/register HTTP/1.1" 201 788
2017-11-24 09:59:45,220 otest.aus.tool:ERROR [Registration] ExcList: Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/oic-0.12.0-py3.5.egg/oic/oauth2/message.py", line 751, in verify
    val = self._dict[attribute]
KeyError: 'redirect_uris'

During handling of the above exception, another exception occurred:
```

**Comment by zandbelt** [comment]
For the record: the root error is about a missing `redirect_uris` entry, but the message shown to the tester is `Missing required attribute "error"`

**Comment by zandbelt** [comment]
this was merged into pyoidc master and should come along with the next release

**Comment by zandbelt** [comment]
part of released 2.0.11
